### ISSUE TITLE
Building this project fails on windows

### ISSUE THREAD

**Issue by justinmchase** [cdglabs/ohm#95]
```
> ohm-js@0.10.0 build-debug e:\code\ohm
> browserify $npm_package_browserify_options

Error: Cannot find module 'e:\code\ohm\$npm_package_browserify_options' from 'e:\code\ohm'
    at e:\code\ohm\node_modules\resolve\lib\async.js:55:21
    at load (e:\code\ohm\node_modules\resolve\lib\async.js:69:43)
    at onex (e:\code\ohm\node_modules\resolve\lib\async.js:92:31)
    at e:\code\ohm\node_modules\resolve\lib\async.js:22:47
    at FSReqWrap.oncomplete (fs.js:82:15)
```

It seems like these shell scripts might be better of as gulp tasks, which works well on windows too. Has there been discussion on that?

**Comment by justinmchase** [comment]
Actually, I don't see where that variable is supposed to be set. Am I missing a step? I just did a clone and `npm install` like in the contributing docs.

**Comment by justinmchase** [comment]
I see it's supposed to be getting the variable from the package itself: https://docs.npmjs.com/misc/scripts#packagejson-vars

I'm not really sure why its working... It could be that since I'm on windows npm things it should be a command instead? I am running it from git bash so it should work but it isn't...

**Comment by pdubroy** [comment]
Hi Justin,

Thanks for helping us get this working on Windows. I'd really prefer not to convert everything to Gulp, if possible. Is it reasonable to assume that most people doing dev on Windows either have Bash installed, or can install it easily? I know back when I was working on Windows I always had it installed, but not sure what things are like these days.

**Comment by justinmchase** [comment]
Yeah I have a PR for this #96. I saw that the script system was fairly complex and agree gulpifting would be a pain.

Also I think it's totally reasonable to expect windows devs to use bash, there are some minor differences in scripts though and can be a bit tricky.

**Comment by pdubroy** [comment]
Fixed by #96.
